client/merchant-feedback-prompt/style.scss

```diff
@@ -29,3 +29,33 @@
 		max-width: 600px;
 	}
 }
+
+.wcpay-merchant-feedback-negative-modal {
+	border-radius: 0 !important;
```

Comment:
Minor thing, I don't think we should worry about modifying minor CSS tweaks like the border-radius to match the design – we should remain consistent across WooPayments with the library version of the modal we're importing.

```suggestion
```